### DISCUSSION TITLE
Column for Field from Relationship (MSSQL)

### DISCUSSION THREAD

**Discussion by zimmra** [Budibase/budibase#5494] [Help]
I've got two tables that I currently have setup in a relationship. 

![image](https://user-images.githubusercontent.com/28514085/164098639-344f5b15-37da-41f6-a6e7-2dd699d3d22c.png)

I am needing to add additional information from the related TimeSheetEntries table to the TimeSheets table.

I tried doing this by adding a formula column to the TimeSheets data table, with the formula `{{ TimeSheetEntries.0.StartTime }}` as an example of one of the fields I am trying to bring over, but that doesn't do anything. 
![image](https://user-images.githubusercontent.com/28514085/164104990-b3d15fa7-4ea8-4f74-bb2c-fbbc0515ba45.png)

**_Am I doing something incorrectly, or is this not possible with SQL Server as the data source?_**

Additional table info:

> #### TimeSheets
>>primary key: Id
>>Created Relationship Column: TimeSheetEntries
>>Desired Related Columns: TimeSheetEntries.StartTime, TimeSheetEntries.EndTime, TimeSheetEntries.Description, TimeSheetEntries.Phase

> #### TimeSheetEntries
>>primary key: Id (does not equal TimeSheets.Id)
>>bb foreign key: TimeSheetID (_does equal_ TimeSheets.Id)
>>Created Relationship Column: TimeSheets
>>Columns to share: StartTime,EndTime,Description,Phase

edit:
After doing some digging I have additional info to add

When setting the formula to `{{ TimeSheetEntries }}` the object is displayed, but essentially all information is missing

The only fields that populate are `primaryDisplay` and `_id`, but as you can see in the screenshot below the helper at least recognizes all of the fields that I am wanting to use

![image](https://user-images.githubusercontent.com/28514085/164145894-ac6c4405-6858-45e6-8984-87e4a859cfc2.png)

**Comment by melohagan** [comment]
Hi @zimmra 

I was able to reproduce this issue, thanks for raising the bug: https://github.com/Budibase/budibase/issues/5495

**Reply by zimmra** [reply]
Thanks for the confirmation @melohagan 

Are the fields supposed to be accessible, or are the helpers just being shown incorrectly?

**Reply by melohagan** [reply]
I'm not sure of the *Define Relationship* inner workings, but I don't see a reason why you shouldn't be able to access related fields. I think this is a bug.

**Reply by zimmra** [reply]
Haven't fully familiarized myself with the inner workings of anything here yet, but seems at least could be related to this section, and the results are being 'squashed'
@melohagan 
https://github.com/Budibase/budibase/blob/27851983edc3bb86cbc41085a85a92e8852f5021/packages/server/src/utilities/rowProcessor/index.js#L253-L291
https://github.com/Budibase/budibase/blob/27851983edc3bb86cbc41085a85a92e8852f5021/packages/server/src/db/linkedRows/index.js#L181-L211
